bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,161 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_nav_platform_cta, vpn_platform_content_well, vpn_platform_content_well_div with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}{{ ftl('vpn-desktop-page-title') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}{{ ftl('vpn-desktop-page-description') }}{% endblock %}
+
+{% block body_id %}mozilla-vpn-platforms-desktop{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-desktop-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
+{% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-platforms') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    custom_nav_cta=vpn_nav_platform_cta(
+      alt_link_text='vpn-shared-cta',
+    )
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block sub_navigation %}
+  {% include 'products/vpn/platforms/includes/subnav-desktop.html' %}
+{% endblock %}
+
+{% block content %}
+<main>
+  <section class="vpn-hero">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="vpn-hero-container">
+        <div class="vpn-hero-body">
+          <h1 class="vpn-hero-title">{{ ftl('vpn-shared-title') }}</h1>
+          <h2 class="vpn-hero-subtitle">{{ ftl('vpn-desktop-hero-headline') }}</h2>
+          <div class="vpn-hero-desc">
+            <p>{{ ftl('vpn-desktop-page-description') }}</p>
+
+            <a class="vpn-button" href="{{ url('products.vpn.landing') }}" data-link-name="Mozilla VPN">{{ ftl('vpn-shared-cta') }}</a>
+
+            <div class="vpn-hero-guarantee">
+                <em>{{ ftl('vpn-shared-day-guarantee') }}</em>
+            </div>
+          </div>
+        </div>
+        <div class="vpn-hero-image hide-from-legacy-ie" data-illustration="n-1">
+          <picture>
+            <source srcset="{{ static('img/products/vpn/platforms/placeholder.png') }} 480w, {{ static('img/products/vpn/platforms/placeholder-high-res.png') }} 960w" media="(min-width: 1024px)">
+            <source srcset="{{ static('img/placeholder.png') }}" media="(max-width: 1023px)">
+            <img src="{{ static('img/products/vpn/platforms/placeholder.png') }}" class="n-1" alt="" width="480" height="480">
+          </picture>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="mzp-l-content mzp-t-content-xl">
+    {% call vpn_platform_content_well_div(
+      title='',
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half vpn-content-well-alt-layout'
+    ) %}
+      <p>{{ ftl('vpn-desktop-ability', devices=5) }}</p>
```

Comment:
Is it worth setting a variable in the template level so you don't need to repeat things like device number etc?

```jinja
{% set vpn_devices = 5 %}
```
and then:

```jinja
{{ ftl('vpn-desktop-ability', devices=vpn_devices) }}
```

Reply:
Done!